jdbc/README.md

```diff
@@ -0,0 +1,16 @@
+Run following in this directory:
+
+```bash
+./run_jdbc_tests.sh <coordinator_port> <jdbc_driver_path>
+```
+
+where,  
+
+coordinator_port: Provide an available port for the coordinator instance of the temporary citus cluster to perform jdbc tests
+jdbc_driver_path: Provide path to jdbc driver for PostgreSQL, (see https://jdbc.postgresql.org/ to download it).
+
+Dependencies:
+* citus_dev
```

Comment:
Should link `[citus_dev](https://github.com/citusdata/tools/tree/develop/citus_dev)`